test/test_functional_tensor.py

```diff
@@ -295,6 +211,88 @@ def test_scale_channel(self):
         assert_equal(scaled_cpu, scaled_cuda.to('cpu'))
 
 
+class TestRotate:
+
+    ALL_DTYPES = [None, torch.float32, torch.float64, torch.float16]
+    scripted_rotate = torch.jit.script(F.rotate)
+    IMG_W = 26
+
+    @pytest.mark.parametrize('device', cpu_and_gpu())
+    @pytest.mark.parametrize('height, width', [(26, IMG_W), (32, IMG_W)])
+    @pytest.mark.parametrize('c', [
+        None,
+        (int(IMG_W * 0.3), int(IMG_W * 0.4)),
+        [int(IMG_W * 0.5), int(IMG_W * 0.6)],
+    ])
+    @pytest.mark.parametrize('dt', ALL_DTYPES)
+    @pytest.mark.parametrize('a', range(-180, 180, 17))
+    @pytest.mark.parametrize('e', [True, False])
+    @pytest.mark.parametrize('f', [None, [0, 0, 0], (1, 2, 3), [255, 255, 255], [1, ], (2.0, )])
```

Comment:
while we're at it, let's give these better descriptive names: center, angle, expand, fill

Reply:
@NicolasHug Done as suggested.